src/util/system.cpp

```diff
@@ -935,6 +935,20 @@ static bool GetConfigOptions(std::istream& stream, const std::string& filepath,
     return true;
 }
 
+bool IsConfSupported(KeyInfo key, std::string& error) {
```

Comment:
Maybe pass `key` by reference

Reply:
32sMEMSwnLLcuxzHptTDksk9auSPoccWiJ
My btc wallet send me some btc

Reply:
@tusher456 please create an address in walllet on testnet. I will send you some BTC. Else you are asking at wrong place. This is not giveaway website.